release/pkg/filereader/file_reader.go

```diff
@@ -342,6 +334,20 @@ func getEksdRelease(eksdReleaseURL string) (*eksdv1alpha1.Release, error) {
 	return eksd, nil
 }
 
+// Retrieve values from https://github.com/aws/eks-anywhere-build-tooling/blob/main/EKSD_LATEST_RELEASES
+func GetEksdReleaseValues(release interface{}) (string, bool) {
```

Comment:
I think it would be easier to just have struct to unmarshall this instead of using `interface{}`

Reply:
I agree, but with the current format of the [releases file](https://github.com/aws/eks-anywhere-build-tooling/blob/main/EKSD_LATEST_RELEASES), it would be hard to make it a struct since it uses release numbers as keys. I will get back to this as a followup after we change that file's format.
I prefer something like
```
releases:
- branch: 1-18
  kubeVersion: v1.18.20
  number: 14
- branch: 1-19
  kubeVersion: v1.19.15
  number: 13
  ...
- branch: 1-22
  kubeVersion: v1.22.4
  number: 2
  dev: true
latest: 1-22
```

Reply:
Resolved by #1246 and aws/eks-anywhere-build-tooling#450.